### DISCUSSION TITLE
Visibility of scheduled jobs (to enqueue the ones I need)

### DISCUSSION THREAD

**Discussion by gabriels1234** [dsoftwareinc/django-rq-scheduler#105] [Ideas]
Hi @cunla,

I still have pending to add the "Enqueue Now" alternative to "Run Now" (scheduling at now() only if it's not scheduled already).
In the meantime, I go to django_rq's scheduled page and enqueue the one I want to be ran now.
The issue is, with the new improvements on the type of args, now it's all more opaque.

<img width="764" alt="image" src="https://user-images.githubusercontent.com/96071233/225880617-a83ecc52-92c3-4b0c-9cce-1b29a5e3fa3f.png">
The detail view gives little to no-help:

<img width="422" alt="image" src="https://user-images.githubusercontent.com/96071233/225885533-cbe8b3cc-4982-47f4-82c9-dd4aa1db067a.png">

Do you think it would make it better if I try to add a feature so that the job_id represents the original callable (like: uuid+callable_name)? (any other idea around making it more user-friendly?)
Thanks!

**Comment by cunla** [comment]
I am currently working on removing the django_rq dependency altogether.
It will take time.
Meanwhile, you can override different views.
